docs/.vuepress/config.js

```diff
@@ -228,7 +228,11 @@ module.exports = {
             title: "Payjoin",
             path: "/Payjoin",
             children: [
-              ["/Payjoin-spec", "Payjoin Specification"]
+              { 
+                title: "Payjoin specification",
+                path: '/Payjoin-spec',
+                redirect: 'https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki'
+              }
```

Comment:
```suggestion
              ["https://github.com/bitcoin/bips/blob/master/bip-0078.mediawiki", "Payjoin specification", { type: 'external' }]
```

Reply:
sorry, I meant the [separate redirects file](https://github.com/btcpayserver/btcpayserver-doc/blob/master/docs/.vuepress/redirects.js) – but let me just add it afterwards there. For the nav we need to specify it as suggested, but the case I meant was for potential links to `/Payjoin-spec` being out there …